spring-batch-docs/asciidoc/spring-batch-integration.adoc

```diff
@@ -948,6 +948,77 @@ when it receives chunks from the master.
 For more information, see the section of the "Scalability" chapter on
 link:$$http://docs.spring.io/spring-batch/reference/html/scalability.html#remoteChunking$$[Remote Chunking].
 
+Starting from version 4.1, Spring Batch Integration provides the `@EnableBatchIntegration`
```

Comment:
I'd love to see your diagram included in the reference documentation.

Reply:
I added the diagram and updated the documentation accordingly